src/routes/firmware.ts

```diff
@@ -20,4 +22,9 @@ router.get('/progress', async (req, res) => {
   res.json(firmwareManager.getProgress());
 });
 
+router.post('/complete', async (req, res) => {
+  exec('reboot'); // Restart the dashcam to apply the new firmware
+  res.json({});
```

Comment:
It's funny it will most probably not even get to this line, be ready for that!

Reply:
true